src/nvim/tui/input.c

```diff
@@ -49,14 +49,9 @@ void tinput_init(TermInput *input, Loop *loop)
   int curflags = termkey_get_canonflags(input->tk);
   termkey_set_canonflags(input->tk, curflags | TERMKEY_CANON_DELBS);
   // setup input handle
-#ifdef WIN32
-  uv_tty_init(&loop->uv, &input->tty_in, 0, 1);
-  uv_tty_set_mode(&input->tty_in, UV_TTY_MODE_RAW);
-  rstream_init_stream(&input->read_stream,
-                      (uv_stream_t *)&input->tty_in,
-                      0xfff);
-#else
   rstream_init_fd(loop, &input->read_stream, input->in_fd, 0xfff);
+#ifdef WIN32
+  uv_tty_set_mode(&input->read_stream.uv.tty, UV_TTY_MODE_RAW);
```

Comment:
why not do this in `stream_init` ?

Reply:
All current calls need to do `uv_tty_set_mode(..., ..., UV_TTY_MODE_RAW)`, so I think that's fine. There may be calls that do not need it, so I did not.